runtime/gc_modron_standard/StandardAccessBarrier.hpp

```diff
@@ -44,7 +43,7 @@
  
 class MM_StandardAccessBarrier : public MM_ObjectAccessBarrier
 {
-private:
+protected:
```

Comment:
this is probably unnecessary, now that Verifier does not have its initialize/teardown